package.json

```diff
@@ -4,7 +4,9 @@
   "description": "first",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon index.js",
+    "jsfmt": "eslint \"src/**/*.js\" --fix",
+    "jslint": "eslint \"src/**/*.js\""
```

Comment:
The linting related code needs to be in a different PR - was this added by default?

Reply:
As a future note, please remember to add `npm` scripts here for the migration and seed so that others can recreate them easily

Reply:
Like npm instead of npx?

Reply:
am not sure about the linting code.